ui-tests/cypress/support/index.js

```diff
@@ -42,3 +42,4 @@ import './restAPI/rolesAPI';
 import './restAPI/usersAPI';
 import './restAPI/userPreferencesAPI';
 import './restAPI/widgetsAPI';
+import './restApi/emailConfigSenderAPI';
```

Comment:
Please, move this import in order to have all the imports in alphabetical order; this way it will be easier to cross-check what has been imported